### PR TITLE
fix: config bug and add Dockerfile for npm publish.

### PR DESCRIPTION
**Pull request by resoliwan**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FROM spearheadea/tsnode:6.9.1-slim-2.1.4
```
Comment (Ohen) [inline Dockerfile]: I think this is oldest image
please reference `island.js`'s Dockerfile
```
FROM spearheadea/tsnode:8.9.4-slim-2.8.4

WORKDIR /app
COPY package.json /app/
COPY .gitignore .npmignore /app/
RUN npm i
RUN ls
COPY spec /app/spec
COPY src /app/src
COPY tsconfig.json gulpfile.js /app/
RUN npm run build
``

Comment (belldoor) [inline Dockerfile]: It seems there is no `*.ts` file and gulp will install from `package.json`  in this project. Why don't you use just pure node-slim image? Here [tsnode image](https://github.com/spearhead-ea/docker-library/blob/master/tsnode/6.9/slim/2.1/Dockerfile) just included typescript, gulp.

Comment (resoliwan) [reply Dockerfile]: I updated to new version.

Comment (resoliwan) [reply Dockerfile]: I think tsnode image is most general image in ACT project.
And this image only used to publish npm package.
The size of image is not important.

**Pull request merged**